.Gruntfile.babel.js

```diff
@@ -253,10 +253,16 @@ module.exports = (grunt) => {
     const cypress = require('cypress')
     const done = this.async()
     const command = grunt.option('browser') === 'debug' ? 'open' : 'run'
+    const disableVideo = process.argv.indexOf('--disable-video') > -1
```

Comment:
Several issues:

1. No need to define a variable, or use `process.argv`, use `grunt.option` instead
2. All video stuff should be disabled by default, and there instead should be a flag to enable it if needed

Reply:
Removed video by default at `cypress.json` config. If we need to add it in a specif case, we just need to edit the file. Simple